5_People/personal-finances.md

```diff
@@ -0,0 +1,117 @@
+
+
+## Optimizing taxes
+
+Niteans are either sole proprietors or have companies that charge Niteo for the work. When you have a registered business, you can legitimately and legally save money on taxes when purchasing things for your company. This can be a computer, phone, desk, chair, software, or anything else you need for your job (and was not paid for directly by Niteo).
+
+You have two options to purchase something: either out-of-pocket from your savings or from your company's pocket. The second option brings you large savings. 
+
+Let's look at an example of a computer purchase. In our first example, we'll buy our computer after we've paid taxes in the company. This tax can be corporate profit tax or, more often, salary taxes.
+
+|     Out-of-pocket         |      | 
+|--------------|------| 
+| Revenue      | $2000 | 
+| Income       | $2000 | 
+| Tax 20%      | $400  | 
+|              |      | 
+| Computer     | $1000 | 
+| Your balance | $600  | 
+
+Now, let's look at an example where we buy the computer as a company expense.
+
+|   Company expense           |      | 
+|--------------|------| 
+| Revenue      | $2000 | 
+| Computer     | $1000 | 
+| Income       | $1000 | 
+| Tax 20%      | $200  | 
+|              |      | 
+| Your balance | $800  | 
+
+As you can see, we've saved $200 by purchasing the computer for the company instead of ourselves. You can do this with all legitimate expenses for your company.
+
+Talk to a local accountant as different jurisdictions have different rules for different types of companies.
+
+## Investing
+
+Investing is a very broad topic and professionals will try to persuade you that you need them to navigate the market. But for the large majority of people investing can be simple and professionals can do more harm than good. 
+
+### Basics of investing
+
+We're assuming that:
+
+- You're investing for the long term (decades, not years, much less months).
+- You don't need the money now or in a year (better yet: only after retirement). 
+- You have the discipline to keep to a strategy even when it looks like it's not working. This means not selling when there is a panic in a market.
+
+When it comes to investing in stocks (company shares) and bonds (loans with fixed interest from governments and corporations), data has shown nobody can beat the market average long-term (decades). Therefore, your goal is to replicate the market as much as possible and at the lowest cost possible. Stock picking, while fun, is closer to the lottery than investing. You might get lucky, but more probably, you'll have less at the end than if you invested in the market. Therefore, we'll be investing in funds that own billions worth of stocks and bonds.
+
+What is the average yearly yield for long-term investing? 7%. It does not sound much but compounded it doubles your money every 10 years. 
```

Comment:
It's closer to 5% when you take the whole World and not just US history, which you should.

https://rationalreminder.ca/podcast/146: `the world index geometric return was 5.2% over this period and the US was 6.6%.`

Reply:
So then it's `1.05^14 = 1.98`, meaning it doubles every 14 years.

Reply:
@zupo Is that after inflation?

Reply:
Updated to 

> What is the average yearly yield for long-term investing? <a href="https://rationalreminder.ca/podcast/146" target="_blank">5.2%</a>. It does not sound much but compounded it doubles your money every 14 years and that's after inflation which would in the same time frame decrease your purchasing power by X%.

I'm researching X.